src/schema/items/task.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: Task
+class: entity
+description: |
+  Each task has a unique label that MUST only consist of letters and/or
+  numbers (other characters, including spaces and underscores, are not
+  allowed).
+  Those labels MUST be consistent across subjects and sessions.
```

Comment:
Similarly to `-echo` entity above, this description is not description of the "task" term, but again for the restriction bids imposes, and specifically to the "entity's label".  So there could be a term "Task" describing what the heck task as a term in general is about. Then `entity-task` (composite) is currently what [TaskName](https://github.com/nqueder/bids_terms_to_markdown_table/blob/master/BIDS_Terms/TaskName.jsonld) is currently. but IMHO  the one which is a composition of a task as a BIDS entity which imposes additional restrictions.  And I do not see ATM any "relationship" to be established between TaskName and TaskDescription -- that they aren't completely independent one from another.  Most likely there should be 3 terms (Task, TaskName, TaskDescription) where the latter two are properties of a Task.